apps/wallet/src/ui/app/pages/home/interstitial/index.tsx

```diff
@@ -76,6 +76,8 @@ function Interstitial({
 					className="flex appearance-none border-none rounded-full bg-[#4CA2FF] h-10 px-6 cursor-pointer items-center no-underline"
 					onClick={onClick}
 					to={bannerUrl}
+					target="_blank"
+					rel="noreferrer noopener"
```

Comment:
Do we actually want to remove the referrer? Sometimes this is used for attribution/analytics. Not sure though if we use this or something else though

Reply:
We have the noreferrer for all the `a` tag link. I will just leave it here for consistency.